Jetcaster/tv/src/main/java/com/example/jetcaster/tv/ui/player/PlayerScreen.kt

```diff
@@ -361,6 +366,12 @@ private fun PlayerControl(
             ElapsedTimeIndicator(timeElapsed, length, skip, rewind)
         }
     }
+    LaunchedEffect(isSafeToSetInitialFocus) {
```

Comment:
We can call `playPauseButton.requestFocus` without checking `isSafeToInitialFocus` flag.